copulas/univariate/gaussian_kde.py

```diff
@@ -133,7 +262,7 @@ def f(x):
 
         return f
 
-    def percent_point(self, U):
+    def percent_point_slow(self, U):
```

Comment:
Remove this after verifying the new method is good to go. Will also require removing helper methods like `_brentq_cdf`.